test/usage/bsb-native/package-lock.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "bsb-native",
+  "version": "0.1.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "bs-platform": {
+      "version": "github:bsansouci/bsb-native#d7e3f615218f0350a2755c4f2a20eb3047ae0dbd",
```

Comment:
When running `npm install`, this triggered

```
npm ERR! code 128
npm ERR! Command failed: /usr/local/bin/git checkout d7e3f615218f0350a2755c4f2a20eb3047ae0dbd
npm ERR! fatal: reference is not a tree: d7e3f615218f0350a2755c4f2a20eb3047ae0dbd
```

It doesn't look like there is such a commit in `bsb-native`. Should `package-lock.json` be checked in at all?